### DISCUSSION TITLE
Laravel 9 Support

### DISCUSSION THREAD

**Discussion by neklein** [spatie/phpunit-watcher#153] [Q&A]
I just wanted to ask, will this package be updated for Laravel 9 support? Right now it cannot be installed due to a symfony/process conflict.

**Comment by freekmurze** [comment]
Feel free to PR the needed changes. 👍

**Reply by neklein** [reply]
Thanks for the response! It looks like the problem is actually with yosymfony/resource-watcher - I'm not sure it is still being maintained - 2 years without a commit and no response to recent pull requests attempts to update for php 8 and symfony 6 support.
